### DISCUSSION TITLE
How can I exit yazi while leaving open tasks running?

### DISCUSSION THREAD

**Discussion by ArijanJ** [sxyazi/yazi#1189] [1. Q&A] [answered]
My use case is that I would like to use yazi to open a PDF, then close yazi while keeping the PDF open, like regular GUI file explorers.

Would I have to override the default open commands for PDFs with i3's `exec` command and have the WM itself start zathura? Not sure if there's a way to achieve this yet, thanks in advance.

**Comment by sxyazi** [comment, marked as answer]
Set `orphan=true` for your opener rule, https://yazi-rs.github.io/docs/configuration/yazi#opener
